Codec/Archive/Tar/Write.hs

```diff
@@ -12,17 +12,21 @@
 -- Portability :  portable
 --
 -----------------------------------------------------------------------------
-module Codec.Archive.Tar.Write (write) where
+module Codec.Archive.Tar.Write (write, write') where
```

Comment:
We probably want to change `Codec.Archive.Tar.create` to use `pack'` and `write'` instead of `pack` and `write`.